packages/core/src/auth/screens/SettingsScreen.js

```diff
@@ -88,6 +90,10 @@ const SettingsScreen = ({route, children}) => {
     [Theme],
   );
 
+  const handleToggleSubtitles = useCallback(() => {
+    setShowSubtitles(!showSubtitles);
+  }, [setShowSubtitles, showSubtitles]);
```

Comment:
this is not optimized, each time the function is called, it will re-render the function as dependencies are modified.
You can get current value of showSubtitles directly is setShowSubtitles function : 

```
  const toggleSubtitles = useCallback(() => {
    setShowSubtitles(_currentState =>!_currentState);
  }, [setShowSubtitles]);
```